emulate/emulate_queue.py

```diff
@@ -139,14 +138,29 @@ def run_code(code: str, duration_of_emulation_in_seconds: int) -> None:
 		logging.info(f'Process finished with return code {return_process_poll}')
 
 
-def retrieve_next_animation() -> Code:
+def retrieve_animation_from_priority_queue() -> Code:
+	priority_queue = PriorityQueue.objects.order_by('priority').last()
+	if priority_queue:
+		priority_code = priority_queue.code
+		priority_queue.delete()
+		return priority_code
+	return None
+
+
+def retrieve_random_animation() -> Code:
 	pk_list = Code.objects.filter(approved=True).values_list('pk', flat=True)
 	if len(pk_list) == 0:
 		raise NoCodeInDatabaseException
 	random_pk = random.choice(pk_list)
 	return Code.objects.get(pk=random_pk)
 
 
+def retrieve_next_animation() -> Code:
+	code_from_priority_queue = retrieve_animation_from_priority_queue()
+	if code_from_priority_queue is not None:
+		return code_from_priority_queue
+	return retrieve_random_animation()
```

Comment:
Ciekawoska pythonowa - the walrus operator

https://docs.python.org/3/whatsnew/3.8.html

Wydaję mi się, że tutaj można to wykorzystać (zaosczedzisz jedną linijke :D)